apps/admin/config/routes.rb

```diff
@@ -4,3 +4,6 @@
 # Example:
 # get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami!']] }
 get '/', to: 'projects#index', as: :projects
+get '/settings', to: 'settings#index', as: :settings
+get '/settings/new', to: 'settings#new', as: :new_settings
+post '/settings/create', to: 'settings#create', as: :create_settings
```

Comment:
could you use [`resources`](http://hanamirb.org/guides/1.1/routing/restful-resources/#remove-routes) here?

Reply:
Of course